FF13Rando/Logic/FF13ItemPlacementLogic.cs

```diff
@@ -92,6 +92,11 @@ public override (string, int)? GetLocationItem(string key, bool orig = true)
                 return t.GetData(orig ? treasureRando.treasuresOrig[key] : treasureRando.treasures[key]);
             case TreasureRando.BattleData b:
                 BattleRando battleRando = treasureRando.Randomizers.Get<BattleRando>();
+                if(battleRando == null)
```

Comment:
stuff like this is a good argument for having the plando functionality for a sub-randomiser in the same class really since a lot of it ends up being duplicated/shared.

Reply:
See comment on the Plandomizer class where we can make an IPlandomizer interface to reuse code and make BattlePlandomizer inherit BattleRandomizer.

That should make it so that `Randomizers.Get<BattleRando>()` pulls whichever type is loaded (normal rando or plando).